sdk/sdk-api/src/main/java/mil/emp3/api/WCS.java

```diff
@@ -25,4 +25,12 @@ public String getServiceURL() {
     public String getCoverageName() {
         return coverageName;
     }
+
+    /**
+     * Prints all configuration parameters
+     */
+    @Override
+    public String toString() {
+        return "URL: " + serviceURL + "\n" + "Coverage: " + coverageName;
+    }
 }
```

Comment:
I see that WCS class has a local member named serviceURL, why do you need that? Base class MapService has that member. Additionally it will be good to validate all the arguments of the constructor and throw appropriate exceptions.